### PR TITLE
Improved ci

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
add coverage and melpa+CI badge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+# ert-runner.el
 
-Ert-runner is a tool for Emacs projects tested using Ert. It assumes a
-certain test structure setup and can therefore make running tests
-easier.
+[![Build Status](https://travis-ci.org/rejeep/ert-runner.el.svg)](https://travis-ci.org/rejeep/ert-runner.el)
+[![Coverage Status](https://coveralls.io/repos/rejeep/ert-runner.el/badge.svg)](https://coveralls.io/r/rejeep/ert-runner.el)
+[![MELPA](http://melpa.org/packages/ert-runner-badge.svg)](http://melpa.org/#/ert-runner)
+[![MELPA Stable](http://stable.melpa.org/packages/ert-runner-badge.svg)](http://stable.melpa.org/#/ert-runner)
+[![Tag Version](https://img.shields.io/github/tag/rejeep/ert-runner.el.svg)](https://github.com/rejeep/ert-runner.el/tags)
```
Comment (lunaryorn) [inline README.md]: I don't know whether we need that badge.  It'll always be the same as MELPA Stable so it seems somewhat redundant to me.  

Also, I'd prefer a different order of badges.  The text reads from left to right, so the most important badges should come first, i.e. left-most.  Most users won't care for our coverage and build status, but only about the version and probably the license.

Hence the MELPA badges should come first, the license badge next, and the status badges at the very end.

**Comment by lunaryorn** [comment]
@AdrieanKhisbe Thank you.  We'll take a look.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,14 @@
-# ert-runner.el [![Build Status](https://api.travis-ci.org/rejeep/ert-runner.el.png?branch=master)](http://travis-ci.org/rejeep/ert-runner.el)
+# ert-runner.el
 
-Ert-runner is a tool for Emacs projects tested using Ert. It assumes a
-certain test structure setup and can therefore make running tests
-easier.
+[![Build Status](https://travis-ci.org/rejeep/ert-runner.el.svg)](https://travis-ci.org/rejeep/ert-runner.el)
+[![Coverage Status](https://coveralls.io/repos/rejeep/ert-runner.el/badge.svg)](https://coveralls.io/r/rejeep/ert-runner.el)
```
Comment (lunaryorn) [inline README.md]: How does coverage work?  Don't we need an account for that?

**Comment by AdrieanKhisbe** [comment]
you need to login to [coveralls](https://coveralls.io/) with your account (any collaborator), and activate coveralls for this repo.

File: README.md (hunk 2 above)
Comment (rejeep) [inline README.md]: I have an account.

File: features/support/env.el [hunk 3]
```diff
@@ -21,6 +21,10 @@
 (defvar ert-runner-output "")
 (defvar ert-runner-error "")
 
+(require 'undercover)
+(undercover "*.el"
+	    (:exclude "*-test.el")
```
Comment (rejeep) [inline features/support/env.el]: We don't have any such files.

File: features/support/env.el [hunk 4]
```diff
@@ -21,6 +21,10 @@
...
+	    (:report-file "/tmp/undercover-report.json"))
```
Comment (rejeep) [inline features/support/env.el]: What's the reason for changing this?

Comment (AdrieanKhisbe) [inline features/support/env.el]: No need in this specific setting indeed.
I just reused the template from cask-package-toolset.

This is needed when you want to merge report from ert and ecukes

**Comment by AdrieanKhisbe** [comment]
> Waaaaaait a minute.

I just realize, cover won't work for the same reason than https://github.com/cask/cask/pull/326#issuecomment-165004614

I'll have to find the motivation to dig into this ^^.
(and probably do an espud+commander task for it to be more reusable)

**Comment by Fuco1** [comment]
Hi @AdrieanKhisbe! Thanks for your contribution, however in the meantime we switched to Github actions (#51) so this is no longer applicable.

**Pull request closed without merging**